src/main/py/peer/wallet.py

```diff
@@ -0,0 +1,67 @@
+"""
+Module to manage peer wallet functionality.
+
+Wallet class allows us to manage the balance of a peer.
+"""
+
+
+class Wallet:
+    """
+    Class to manage Peer wallet functionality
+    """
+
+    def __init__(self):
+        """
+        Initializes an instance of the Wallet class
+        """
+
+        self.balance = 0
+        self.pending_balance = 0
+
+    def add_to_balance(self, amount: int):
+        """
+        Adds balance to the wallet.
+
+        Params:
+        - amount (int): The amount to add to the wallet.
+        """
+
+        self.balance += amount
+
+    def remove_from_balance(self, amount: int):
+        """
+        Removes balance from the wallet.
+
+        Params:
+        - amount (int): The amount to add to the wallet.
+        """
+
+        self.balance -= amount
+
+    def add_pending_balance(self, amount: int):
+        """
+        Adds pending balance to the wallet.
+
+        Params:
+        - amount (int): The amount to add to the pending balance.
+        """
+
+        self.pending_balance += amount
+
+    def remove_pending_balance(self, amount: int):
```

Comment:
Thanks for `remove_from...` above, but this was missed... (original comment was "here and the other methods like this").